Separate LKE Cluster Details and Resize into two nested tabs: Details Page

WilkinsKa1 opened the pull request:
## Description

Please note this includes work from this [PR](https://github.com/linode/manager/pull/5728) so that needs to be merged first. This PR focuses on the /details tab, which includes a modified "view only" version of node pools details and a documentation blurb. 

## Type of Change
- Non breaking change ('change')

Jskobos commented on packages/manager/src/features/Kubernetes/KubernetesClusterDetail/NodePoolsDisplay/NodePoolsDisplay.tsx:
```diff
@@ -85,18 +85,23 @@ const styles = (theme: Theme) =>
-  deletePool: (poolID: number) => void;
-  resetForm: () => void;
-  submitForm: () => void;
+  // updatePool: (poolIdx: number, updatedPool: PoolNodeWithPrice) => void;
```
We should probably (later) separate NodePoolsDisplay into two components, now that the same view won't render both editing/non-editing versions. That way we can have the props be more precise. For now probably just delete the commented out bits.

Jskobos commented on packages/manager/src/features/Kubernetes/KubernetesClusterDetail/Details.tsx:
```diff
@@ -1,7 +1,117 @@
+  }
+
+  /** Holds the local state of the cluster's node pools when editing */
+  const [pools] = React.useState<PoolNodeWithPrice[]>(cluster.node_pools || []);
```
This isn't needed, you can delete these lines and then just pass `poolsForEdit={[]}` below.

WilkinsKa1 commented:
@Jskobos Thanks for reviewing, I addressed your feedback here: https://github.com/linode/manager/pull/5732 bc of rebase issues.

Closing in favor of https://github.com/linode/manager/pull/5732
